app/views/child_benefit_tax/_part_tax_year_conditional.html.erb

```diff
@@ -25,10 +25,13 @@
     margin_bottom: 4,
     font_size: 24
   } %>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>you can find the start date on your Child Benefit award notice</li>
-    <li>the stop date is usually when a child turns 16 or leaves full-time education</li>
-  </ul>
+  <%= render "govuk_publishing_components/components/list", {
+      visible_counters: true,
+      items: [
+          sanitize("you can find the start date on your Child Benefit award notice"),
```

Comment:
Since this content is static, does it need sanitising? Nothing wrong with it I guess but the intent if sanitising is to catch malicious html so I don't know if this necessary. Mostly just a code tidiness and tiny little processing weight thing.

Reply:
Ah course, good spot - removed.